.azure-pipelines/MacOS-CI.yml

```diff
@@ -10,13 +10,34 @@ jobs:
       py27:
         python.version: '2.7'
         onnx_ml: 0
+        onnx_debug: 0
+        onnx_lite: 'OFF'
+      py27-ml:
+        python.version: '2.7'
+        onnx_ml: 1
+        onnx_debug: 0
+        onnx_lite: 'OFF'
+      py27-ml-lite:
```

Comment:
python 2.7 is going to be deprecated soon.. can you simply add the LITE option to Py36-ml?

Reply:
added!